### PR TITLE
 Added type definitions for withings-lib

### PR DESCRIPTION
**Pull request by mpicciolli**
case 1. Add a new type definition.
- [ ] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts`

### CODE REVIEW

File: withings-lib/withings-lib.d.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+// Definitions by: mpicciolli <https://github.com/mpicciolli>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare var WithingsClient:WithingsClient;
+
+declare module 'withings-lib' {
+    export = WithingsClient;
+}
+
+interface WithingsClient {
```
Comment (vvakame) [inline withings-lib/withings-lib.d.ts]: http://definitelytyped.org/guides/best-practices.html#ghost-modules
please check this document.

**Comment by mhegazy** [comment]
@mpicciolli can you update the PR to use the ghost module pattern?

**Comment by mhegazy** [comment]
@mpicciolli 🔔

**Comment by jramsay** [comment]
@mpicciolli: Thanks for your contribution. This PR has unaddressed comments and cannot be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**